lib/please_cli/please_cli/terraform_route53_config.py

```diff
@@ -122,16 +131,62 @@ def cmd(channel):
                     domain = domain.lstrip('http')
                     domain = domain.lstrip('://')
 
-                if project_target == 'HEROKU':
-                    click.echo(HEROKU_TEMPLATE % dict(
-                        name=to_route53_name(project_id, channel),
-                        domain=domain,
-                        dns=project[channel]['dns'],
-                    ))
+                def heroku(a, b):
+
+                    if project_target == 'HEROKU' and a == channel and 'shipit-' not in project_id:
+                        b.append(HEROKU_TEMPLATE % dict(
+                            name=to_route53_name(project_id, channel),
+                            domain=domain,
+                            dns=project[channel]['dns'],
+                         ))
+
+                def shipit(a, b):
+
+                    if project_target == 'HEROKU' and a == channel and 'shipit-' in project_id:
+                        b.append(HEROKU_TEMPLATE % dict(
+                            name=to_route53_name(project_id, channel),
+                            domain=domain,
+                            dns=project[channel]['dns'],
+                         ))
+
+                def comment(a):
+                    line = '#' * len(HEROKU_COMMENT.format(channel=a))
+                    click.echo(line)
+                    click.echo(HEROKU_COMMENT.format(channel=a))
+                    click.echo(line)
 
-                elif project_target == 'S3':
+                heroku('production', HEROKU_PRODUCTION)
+                heroku('testing', HEROKU_TESTING)
+                heroku('staging', HEROKU_STAGING)
+
+                shipit('production', SHIPIT_PRODUCTION)
+                shipit('testing', SHIPIT_TESTING)
+                shipit('staging', SHIPIT_STAGING)
+
+                if project_target == 'S3':
                     click.echo(S3_TEMPLATE % dict(
                         name=to_route53_name(project_id, channel),
                         domain=domain,
                         dns=project[channel]['dns'],
                     ))
+
+    comment('production')
+    click.echo(pprint.pprint(HEROKU_PRODUCTION))
```

Comment:
it would be better to loop over HEROKU_PRODUCTION and use HEROKU_TEMPLATE for each. also please sort before printing (eg. ``sorted(HEROKU_PRODUCTION, key=lambda x: x['name'])``)